Jetsurvey/app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.compose.jetsurvey">
 
+    <uses-permission android:name="android.permission.CAMERA"/>
```

Comment:
I don't think you need this permission. You can trigger the intent without requesting it

Reply:
Pretty sure this is still required, but I'm not 100% sure

Reply:
I checked and it's not useful: https://developer.android.com/training/permissions/evaluating#take-photo

Reply:
That only works if you use the use the intent to open the Camera app. This PR is all about requesting permissions, and this line is needed to demonstrate that AFAIK.

Reply:
Yacine is right, this question uses the `TakePicture` activity result contract that does NOT require the camera permission. I thought it was needed (well, at least, I tested the flow with more permissions :D haha)

The `WRITE_EXTERNAL_STORAGE` is still required for APIs lower than 29 so we'd be showcasing permissions anyway.